onnxruntime/python/onnxruntime_pybind_mlvalue.cc

```diff
@@ -37,11 +38,19 @@ static bool PyObjectCheck_NumpyArray(PyObject* o) {
   return PyObject_HasAttrString(o, "__array_finalize__");
 }
 
+bool IsNumpyArray(py::object& obj) {
```

Comment:
const ?

Reply:
Well. I tried my best, but I could not do everything. See here: [https://pybind11.readthedocs.io/en/stable/limitations.html#design-choices](https://pybind11.readthedocs.io/en/stable/limitations.html#design-choices)